src/CentreonCommand/Domain/Repository/CommandRepository.php

```diff
@@ -88,7 +88,7 @@ public function getPaginationList(
         $collector->bind($stmt);
 
         $stmt->execute();
-        $stmt->setFetchMode(PDO::FETCH_CLASS, Command::class);
+        $stmt->setFetchMode(PDO::FETCH_CLASS, (object)Command::class);
```

Comment:
2nd parameter of setFetchMode with FETCH_CLASS options should be a string 
https://www.php.net/manual/en/pdostatement.setfetchmode.php

`public PDOStatement::setFetchMode(int $mode = PDO::FETCH_CLASS, string $class, ?array $constructorArgs): bool`

Reply:
removed (object)